### PR TITLE
Random bits and pieces

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

[Improve the look of the image gallery](https://github.com/webkom/lego-webapp/commit/eaf62c0a45e5df984f88b6cd587032258c4f901c)

<table>
	<tr>
		<td>Before</td>
		<td>After</td>
	<tr>
		<td>
			
		</td>
		<td>
			
		</td>
</table>

## Result

---

[Misc updates to comments](https://github.com/webkom/lego-webapp/commit/1061c82737e355a4aa0e5b66993da276576b2c3c) 

* Profile picture is now also a link
* Comments made by deleted users now look "normal"

## Result

<table>
	<tr>
		<td>Before</td>
		<td>After</td>
	<tr>
		<td>
			
		</td>
		<td>
			
		</td>
</table>

---

[Clean up misc things to bdb](https://github.com/webkom/lego-webapp/commit/d888d9b35a87912c707ea7dd77ab74c5c8bf9ad4) 

Among other things; link icons previously did not get the proper styling.

## Result

<table>
	<tr>
		<td>Before</td>
		<td>After</td>
	<tr>
		<td>
			image
		</td>
		<td>
			image
		</td>
</table>

---

[Update and fix random bnp](https://github.com/webkom/lego-webapp/commit/a090ba391a4ff8279745f5ebd594c57a6754ef4f)

## Result

A more fitting icon and text imo:

<table>
	<tr>
		<td>Before</td>
		<td>After</td>
	<tr>
		<td>
			image
		</td>
		<td>
			image
		</td>
</table>

# Testing

- [x] I have thoroughly tested my changes.

Not really any breaking changes, but everything that was changed was tested. I specifically tested that the changes to the icon component did not break anything.

### CODE REVIEW

File: app/routes/events/components/RegisteredSummary.tsx [hunk 1]
```diff
@@ -119,7 +119,7 @@ const RegisteredSentence = ({
 const RegisteredSummary = (props: RegisteredSummaryProps) => {
   return (
     <Flex className={styles.summary}>
-      <RegisteredSentence {...props} /> er påmeldt.
```
Comment (ollfkaih) [inline app/routes/events/components/RegisteredSummary.tsx:122]: I liked that little guy :/

File: app/routes/bdb/components/BdbDetail.tsx [hunk 2]
```diff
@@ -246,16 +235,15 @@ export default class BdbDetail extends Component<Props, State> {
             <td>{truncateString(event.description, 70)}</td>
             <td>
               {event.survey && (
-                <Tooltip content="Spørreundersøkelse">
-                  <Link to={`/surveys/${event.survey}`}>
-                    <i
-                      className={cx(
```
Comment (ollfkaih) [inline app/routes/bdb/components/BdbDetail.tsx:252]: surely your ide could auto-remove the import?

**Review by ollfkaih — APPROVED** [review]
looks good to me

**Review by Arashfa0301 — APPROVED** [review]
Looks good to me 💯

**Pull request merged**